### PR TITLE
Use javascript and CSS .map files for debuginfo

### PR DESCRIPTION
**Pull request by stefwalter**
This is a whole bunch of changes which change our debuginfo to use the map files that webpack and other tools generate. This brings them much more inline with the debuginfo we include for binaries.

### CODE REVIEW

**Comment by stefwalter** [comment]
Blocked on #5182

**Comment by stefwalter** [comment]
Rebased.

**Comment by dperpeet** [comment]
build seems to fail everywhere

File: lib/patternfly/dist/css/patternfly-additions.css [hunk 1]
```diff
@@ -2885,7 +2885,8 @@ table.datatable th:active {
 }
 @font-face {
   font-family: 'PatternFlyIcons-webfont';
-  src: url('fonts/patternfly.woff') format('woff');
+  src: url('../fonts/PatternFlyIcons-webfont.eot');
+  src: url('../fonts/PatternFlyIcons-webfont.eot?#iefix') format('embedded-opentype'), url('../fonts/PatternFlyIcons-webfont.ttf') format('truetype'), url('../fonts/PatternFlyIcons-webfont.woff') format('woff'), url('../fonts/PatternFlyIcons-webfont.svg#PatternFlyIcons-webfont') format('svg');
```
Comment (dperpeet) [inline lib/patternfly/dist/css/patternfly-additions.css]: as stated on irc: I don't understand why all these urls need to be on the same line

Comment (stefwalter) [inline lib/patternfly/dist/css/patternfly-additions.css]: Upstream in patternfly they are on teh same line ... so you might choose to file a bug there.

That said, this pull request should not change any lines in this file. Thanks for catching this.

Comment (stefwalter) [inline lib/patternfly/dist/css/patternfly-additions.css]: I've verified that the patternfly CSS installed and distributed by Cockpit does not have this multi url line. It merely shows up here as the original file, which is amended during the build with our sed script.

Comment (dperpeet) [inline lib/patternfly/dist/css/patternfly-additions.css]: ok

**Pull request closed without merging**